SICP_book/chapter_1_2.metta

```diff
@@ -0,0 +1,645 @@
+; Extension is used for several examples which requires random and current-inexact-milliseconds functions from Scheme
+!(extend-py! additional_funcs)
+
+(= (inc $x) (+ $x 1))
+(= (dec $x) (- $x 1))
+
+(= (sqr $x) (* $x $x))
+
+(= (>= $x $y) (or (> $x $y) (== $x $y)))ex
```

Comment:
1. typo:
```suggestion
(= (>= $x $y) (or (> $x $y) (== $x $y)))
```
2. `>=` and `<=` are also present in a standard library, not sure is you need them implemented independently

Reply:
Ok, I'll delete this.